src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -30,25 +29,33 @@ const styles = {
 const groupCallStatuses = _.flow(
   _.values,
   _.flattenDepth(1),
-  _.countBy(a => collapseCromwellExecutionStatus(a.executionStatus))
+  _.countBy(a => {
+    const collapsedStatus = collapseCromwellStatus(a.executionStatus, a.backendStatus)
+    return collapsedStatus !== statusType.unknown ? collapsedStatus.id : collapsedStatus.label(a.executionStatus)
+  })
 )
 
 const statusCell = ({ calls }) => {
-  const { succeeded, failed, running, submitted, ...others } = groupCallStatuses(calls)
+  // Note: these variable names match the id values of statusType (except for others, which will be the labels for unknown statuses).
+  const { succeeded, failed, running, submitted, waitingForQuota, ...others } = groupCallStatuses(calls)
 
-  const makeRow = (count, icon, text) => {
+  const makeRow = (count, status, labelOverride) => {
+    const seeMore = !!status.moreInfoLink ? h(Link, { href: status.moreInfoLink, style: { marginLeft: '0.50rem' }, ...Utils.newTabLinkProps },
+      [status.moreInfoLabel, icon('pop-out', { size: 12, style: { marginLeft: '0.25rem' } })]) : ''
     return !!count && div({ style: { display: 'flex', alignItems: 'center', marginTop: '0.25rem' } }, [
-      icon,
-      ` ${count} ${text}`
+      status.icon(),
+      ` ${count} ${!!labelOverride ? labelOverride : status.label()}`,
+      seeMore
     ])
   }
 
   return h(Fragment, [
-    makeRow(submitted, submittedIcon(), 'Submitted'),
-    makeRow(running, runningIcon(), 'Running'),
-    makeRow(succeeded, successIcon(), 'Succeeded'),
-    makeRow(failed, failedIcon(), 'Failed'),
-    _.map(([name, count]) => makeRow(count, unknownIcon(), name), _.toPairs(others))
+    makeRow(submitted, statusType.submitted),
+    makeRow(waitingForQuota, statusType.waitingForCloudQuota),
+    makeRow(running, statusType.running),
+    makeRow(succeeded, statusType.succeeded),
+    makeRow(failed, statusType.failed),
+    h(Fragment, _.map(([label, count]) => makeRow(count, statusType.unknown, label), _.toPairs(others)))
```

Comment:
Note the `Fragment` usage here to avoid this issue: `index.js:1 Warning: Each child in a list should have a unique "key" prop.`. `._map` returns an array.